unit_tests/test_aggregator.py

```diff
@@ -294,84 +262,78 @@ def test_calculate_grid_fee__return(aggregator_explicit):
 
 
 class TestExecuteBatchCommands:
-    """Includes methods which test execute_batch_commands method."""
+    """Test the execute_batch_commands behaviors."""
 
     @staticmethod
-    def test_execute_batch_commands__command_buffer_length_zero(aggregator_explicit):
+    def test_execute_batch_commands_command_buffer_length_zero(aggregator_explicit):
         with patch("gsy_e_sdk.aggregator.ClientCommandBuffer.buffer_length",
                    new_callable=PropertyMock, return_value=0):
             assert aggregator_explicit.execute_batch_commands() is None
 
     @staticmethod
-    def test_execute_batch_commands__execute_batch_called(aggregator_explicit):
-        with patch("gsy_e_sdk.aggregator.ClientCommandBuffer.execute_batch") as mocked_method:
-            aggregator_explicit.start_websocket_connection()
-            aggregator_explicit.execute_batch_commands()
-            mocked_method.assert_called_once()
+    def test_execute_batch_commands_execute_batch_called(aggregator_explicit):
+        aggregator_explicit.start_websocket_connection()
+        aggregator_explicit.execute_batch_commands()
+        aggregator_explicit._client_command_buffer.execute_batch.assert_called_once()
```

Comment:
We can directly access the `_client_command_buffer` interface even if it's private, it probably is more straightforward than patching the object

Reply:
This would work because the `execute_batch` method has already been patched in `mock_execute_batch_command_methods` fixture, right?

Reply:
100% no need to patch it again here